src/hott/03-equivalences.rzk.md

```diff
@@ -271,11 +271,11 @@ invertible map to prove symmetry:
   ( e : Equiv A B)
   : Equiv B A
   :=
-    ( first (has-inverse-is-equiv A B (first e) (second e)) ,
-      ( ( first e ,
-          second (second (has-inverse-is-equiv A B (first e) (second e)))) ,
-        ( first e ,
-        first (second (has-inverse-is-equiv A B (first e) (second e))))))
+    ( first (has-inverse-is-equiv A B (first e) (second e))
+    , ( ( first e
+        , second (second (has-inverse-is-equiv A B (first e) (second e))))
+      , ( first e
+      , first (second (has-inverse-is-equiv A B (first e) (second e))))))
```

Comment:
This one's misplaced. It should be one level more indented.

Reply:
The last line, that is

Reply:
This is due to incorrect indentation in the original code. The formatter does not fix that (at least not at the moment).